cob_hardware_config/components/sick_s300/sick_s300.yaml

```diff
@@ -1,9 +1,4 @@
-port: /dev/ttyUSB0
 baud: 500000
 scan_duration: 0.025 #no info about that in SICK-docu, but 0.025 is believable and looks good in rviz
 scan_cycle_time: 0.040 #SICK-docu says S300 scans every 40ms
-inverted: false
-scan_id: 7
-frame_id: base_laser_rear_link
 publish_frequency: 12
-scan_intervals: [[-2.0, 2.0]] #[rad] these intervals are included to the scan
```

Comment:
I'm still not sure if it makes sense to have those parameters only "component-specific". The ones you pull out are definitely depending on the real instance of the sensor. The rest could, however, in theory be changed as well.

What I don't like too much is that we now have the parameters in two places. The yaml file here and the launch args in the respective modules. This probably makes sense for many nodes, but especially for this one, one could argue that if those cannot/should not be changed depending on the real instance, they could/should be hard-coded and not parameters at all.

I fully understand the idea behind this, but I think there are cases where it is difficult to decide what is depending on the real sensor and what on the sensor type...